pinhook/plugin.py

```diff
@@ -47,6 +58,7 @@ def clear_plugins():
 
 
 def register(command, help_text=None):
+    @wraps(command)
```

Comment:
What functionality does this decorator add?

Reply:
So, I read the docs, and this is actually cool. So that it retains the name of the original function.